src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -646,7 +619,7 @@ def setUp(self):
 
         # Access points to use by default when starting servers and agents
         self.access_points = self.params.get(
-            "access_points", "/run/setup/*", self.hostlist_servers[:1])
+            "access_points", "/run/setup/*", list(self.hostlist_servers)[:1])
 
         # Display host information
         self.log.info("-" * 100)
```

Comment:
Can the Display output alos be NodeSet?
        # Display host information
        self.log.info("-" * 100)
        self.log.info("--- HOST INFORMATION ---")
        self.log.info("hostlist_servers:    %s", str(NodeSet.fromlist(self.hostlist_servers))
        etc....

Reply:
@mjean308 do you have a specific example?  The `HOST INFORMATION` is being displayed as a `NodeSet`:
```
2022-04-27 10:36:36,225 test             L0626 INFO | --- HOST INFORMATION ---
2022-04-27 10:36:36,225 test             L0627 INFO | hostlist_servers:    wolf-104vm[2-6]
2022-04-27 10:36:36,225 test             L0628 INFO | hostlist_clients:    
2022-04-27 10:36:36,225 test             L0629 INFO | server_partition:    None
2022-04-27 10:36:36,225 test             L0630 INFO | client_partition:    None
2022-04-27 10:36:36,225 test             L0631 INFO | server_reservation:  None
2022-04-27 10:36:36,225 test             L0632 INFO | client_reservation:  None
2022-04-27 10:36:36,226 test             L0633 INFO | access_points:       ['wolf-104vm2']
```

Reply:
@phender this is what is looks like on sunspot
2022-05-02 02:18:04,467 test             L0648 INFO | ----------------------------------------------------------------------------------------------------
2022-05-02 02:18:04,467 test             L0649 INFO | --- HOST INFORMATION ---
2022-05-02 02:18:04,467 test             L0650 INFO | hostlist_servers:    ['daos-0013', 'daos-0014', 'daos-0015', 'daos-0016', 'daos-0017', 'daos-0018', 'daos-0019', 'daos-0020']
2022-05-02 02:18:04,469 test             L0651 INFO | hostlist_clients:    ['daos-0001', 'daos-0002', 'daos-0003', 'daos-0004', 'daos-0005', 'daos-0006', 'daos-0007', 'daos-0008', 'daos-0009', 'daos-0010', 'daos-0011', 'daos-0012', 'daos-0013', 'daos-0014', 'daos-0015', 'daos-0016', 'daos-0017', 'daos-0018', 'daos-0019', 'daos-0020']
2022-05-02 02:18:04,469 test             L0652 INFO | server_partition:    None
2022-05-02 02:18:04,469 test             L0653 INFO | client_partition:    daos_client
2022-05-02 02:18:04,469 test             L0654 INFO | server_reservation:  None
2022-05-02 02:18:04,470 test             L0655 INFO | client_reservation:  None
2022-05-02 02:18:04,470 test             L0656 INFO | access_points:       ['daos-0013']
2022-05-02 02:18:04,470 test             L0659 INFO | ----------------------------------------------------------------------------------------------------
2022-05-02 02:18:04,470 test             L0660 DEBUG| Common test directory (/var/tmp/daos_testing/test_soak_stress) contents:
2022-05-02 02:18:05,309 test             L0677 INFO | ----------------------------------------------------------------------------------------------------

Reply:
Something like this would be easier to read when we are dealing with 1024 nodes

--- HOST INFORMATION ---
hostlist_servers: daos-[0013-0020]
hostlist_clients: daos-[0001-0020]

Reply:
soak/smoke.py passed in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8809/10/artifact/Functional%20Hardware%20Large/soak/smoke.py and produced this output:
```
2022-05-29 18:08:59,166 process          L0604 INFO | Running 'scontrol show partition daos_client'
2022-05-29 18:08:59,172 process          L0416 DEBUG| [stdout] PartitionName=daos_client
2022-05-29 18:08:59,172 process          L0686 INFO | Command 'scontrol show partition daos_client' finished with 0 after 0.004477500915527344s
2022-05-29 18:08:59,172 process          L0416 DEBUG| [stdout]    AllowGroups=ALL AllowAccounts=ALL AllowQos=ALL
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    AllocNodes=ALL Default=YES QoS=N/A
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    DefaultTime=NONE DisableRootJobs=NO ExclusiveUser=NO GraceTime=0 Hidden=NO
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    MaxNodes=UNLIMITED MaxTime=UNLIMITED MinNodes=0 LLN=NO MaxCPUsPerNode=UNLIMITED
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    Nodes=wolf-[142-149]
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    PriorityJobFactor=1 PriorityTier=1 RootOnly=NO ReqResv=NO OverSubscribe=NO
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    OverTimeLimit=NONE PreemptMode=OFF
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    State=UP TotalCPUs=576 TotalNodes=8 SelectTypeParameters=NONE
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    JobDefaults=(null)
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout]    DefMemPerNode=UNLIMITED MaxMemPerNode=UNLIMITED
2022-05-29 18:08:59,173 process          L0416 DEBUG| [stdout] 
2022-05-29 18:08:59,174 write_host_file  L0040 INFO | Writing hostfile:
  hosts: wolf-[142-149]
  slots: 1
  file:  /var/tmp/avocado_2qcpn5ox/avocado_job_4if2c9xx/1-._soak_smoke.py_SoakSmoke.test_soak_smo_run-container-container_reserved-daos_racer-fio_smoke-dfuse-global-soak-test-hdf5_vol-hosts-ior_smoke-macsio_smoke-mdtest_smoke-orterun-pool_jobs-pool_racer-pool_reserved-server_config-servers-0-1-setup-smoke-dd5e/hostfile76378
2022-05-29 18:08:59,175 parameters       L0143 DEBUG| PARAMS (key=access_points, path=/run/setup/*, default=['wolf-142']) => ['wolf-142']
2022-05-29 18:08:59,175 test             L0706 INFO | ----------------------------------------------------------------------------------------------------
2022-05-29 18:08:59,175 test             L0707 INFO | --- HOST INFORMATION ---
2022-05-29 18:08:59,175 test             L0708 INFO | hostlist_servers:    wolf-[142-145]
2022-05-29 18:08:59,175 test             L0709 INFO | hostlist_clients:    wolf-[142-149]
2022-05-29 18:08:59,175 test             L0710 INFO | server_partition:    None
2022-05-29 18:08:59,175 test             L0711 INFO | client_partition:    daos_client
2022-05-29 18:08:59,175 test             L0712 INFO | server_reservation:  None
2022-05-29 18:08:59,175 test             L0713 INFO | client_reservation:  None
2022-05-29 18:08:59,175 test             L0714 INFO | access_points:       ['wolf-142']
2022-05-29 18:08:59,175 test             L0717 INFO | ----------------------------------------------------------------------------------------------------
```